### PR TITLE
:rocket: add option to provide week days to display in week.

### PR DESCRIPTION
**Pull request by ParthBaraiya**

### CODE REVIEW

**Comment by sanket-simform** [comment]
I think let's add one more property showWeekend and defaut it to true along with combination of this weekdays array.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 0.0.3
+
+- Add support to choose which days should be displayed in a week in `WeekView`.
```
Comment (vatsaltanna-simformsolutions) [inline CHANGELOG.md:3]: please mention the issue here

File: lib/src/extensions.dart [hunk 2]
```diff
@@ -45,9 +43,9 @@ extension DateTimeExtensions on DateTime {
   /// will return dates
   /// [6,7,8,9,10,11,12]
   /// Where on 6th there will be monday and on 12th there will be Sunday
-  List<DateTime> datesOfWeek({WeekDays startDay = WeekDays.monday}) {
+  List<DateTime> datesOfWeek() {
     final day = weekday;
-    final start = subtract(Duration(days: day - startDay.index));
+    final start = subtract(Duration(days: day - 1));
```
Comment (vatsaltanna-simformsolutions) [inline lib/src/extensions.dart:48]: why are we decreasing 1 day here ?

File: lib/src/week_view/week_view.dart [hunk 3]
```diff
@@ -264,7 +313,7 @@ class WeekViewState<T> extends State<WeekView<T>> {
                     onPageChanged: _onPageChange,
                     itemBuilder: (_, index) {
                       final dates = _minDate
-                          .add(Duration(days: (index - 1) * _weekDays))
+                          .add(Duration(days: (index - 1) * 7))
```
Comment (vatsaltanna-simformsolutions) [inline lib/src/week_view/week_view.dart:316]: use constant variables instead of direct value

**Pull request merged**